src/vs/workbench/contrib/webview/browser/webviewEditor.ts

```diff
@@ -178,8 +179,33 @@ export class WebviewEditor extends BaseEditor {
 			this._content.setAttribute('aria-flowto', input.webview.container.id);
 		}
 
+		this._webviewVisibleDisposables.clear();
+
+		// Webviews are not part of the normal editor dom, so we have to register our own drag and drop handler on them.
+		this._webviewVisibleDisposables.add((this._editorGroupsService as EditorPart).createDropTargets(input.webview.container, {
```

Comment:
@mjbvz I think one step further is to make this new method available from `IEditorGroupsService` to prevent this ugly cast.

Reply:
Not sure if this is possible. `IEditorGroupsService` is in common so it can't reference `HTMLElement`. I've replaced the cast with an instanceof check which should be safer

Reply:
I filed https://github.com/microsoft/vscode/issues/82968 to follow up.